source/features/linkify-user-location.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {wrap} from '../libs/dom-utils';
+import features from '../libs/features';
+
+function addLocation(baseElement: HTMLElement): void {
+	for (const {nextElementSibling, nextSibling} of select.all('.octicon-location', baseElement)) {
+		const location = nextElementSibling || nextSibling!;
+
+		if (!nextElementSibling) {
+			location.before(document.createTextNode(' '));
+		}
```

Comment:
You can avoid the condition altogether. An extra space there even if isn't necessary doesn't hurt anyone. Also `.before` accepts plain strings :tada:

```js
location.before(' ');
```

Other than this, I think the rest looks great! Good call on df14c51ebca9ed0bcc07b7430f57b9506b34bf50